src/textual/reactive.py

```diff
@@ -169,53 +159,41 @@ def _reset_object(cls, obj: object) -> None:
         getattr(obj, "__watchers", {}).clear()
         getattr(obj, "__computes", []).clear()
 
-    def __set_name__(self, owner: Type[MessageTarget], name: str) -> None:
-
+    def __set_name__(self, owner: Reactable, name: str) -> None:
         # Check for compute method
         if hasattr(owner, f"compute_{name}"):
             # Compute methods are stored in a list called `__computes`
-            try:
-                computes = getattr(owner, "__computes")
-            except AttributeError:
+            if not hasattr(owner, "__computes"):
                 computes = []
                 setattr(owner, "__computes", computes)
+            else:
+                computes = getattr(owner, "__computes")
             computes.append(name)
 
         # The name of the attribute
         self.name = name
         # The internal name where the attribute's value is stored
         self.internal_name = f"_reactive_{name}"
-        default = self._default
-        setattr(owner, f"_default_{name}", default)
 
     def __get__(self, obj: Reactable, obj_type: type[object]) -> ReactiveType:
         _rich_traceback_omit = True
 
         self._initialize_reactive(obj, self.name)
-
-        value: ReactiveType
-        compute_method = getattr(self, f"compute_{self.name}", None)
-        if compute_method is not None:
-            old_value = getattr(obj, self.internal_name)
-            value = getattr(obj, f"compute_{self.name}")()
-            setattr(obj, self.internal_name, value)
-            self._check_watchers(obj, self.name, old_value)
-        else:
-            value = getattr(obj, self.internal_name)
+        value: ReactiveType = getattr(obj, self.internal_name)
```

Comment:
works both way (`name` and `internal_name`), but `internal_name` is the original meaning I believe. `name` works but is more roundabout, plus before I deleted some more code below was harder to reason about. And contained dead code, which I originally thought was a bug.